CHANGELOG.md

```diff
@@ -33,7 +33,8 @@ The general format is:
 
 ### Added
 
-- ability to set the `access_modes` for dynamically privisioned volumes 
+- ability to set the `access_modes` for dynamically privisioned volumes
+- support for `env_from` option
```

Comment:
Hera follows semantic versioning so, for a new feature, I suggest we bump the minor version, which means adding a new entry in the change log! So, I think the entry `- support for `env_from` option` should move to a block that looks like: 
```
# 2.4.0 - DATE (27/03/2022)

### Added

- support for `env_from` option
```

Happy to provide more information about semantic versioning! There are some more details in the [contribution guide](https://github.com/argoproj-labs/hera-workflows/blob/main/CONTRIBUTING.md)!

Reply:
Thank you for the review!

I fixed `CHANGELOG.md` and upgrade `VERSION`